model.py

```diff
@@ -7953,8 +7953,8 @@ class CustomList(Base):
     id = Column(Integer, primary_key=True)
     primary_language = Column(Unicode, index=True)
     data_source_id = Column(Integer, ForeignKey('datasources.id'), index=True)
-    foreign_identifier = Column(Unicode, index=True)
-    name = Column(Unicode)
+    foreign_identifier = Column(Unicode, index=True, unique=True)
+    name = Column(Unicode, index=True, unique=True)
```

Comment:
I'm not sure about this choice, but in the script I'm creating `foreign_identifier` based on name. I don't want there to be any confusion around this in the future...